skyvern/webeye/actions/handler.py

```diff
@@ -828,9 +820,16 @@ async def normal_select(
     action: actions.SelectOptionAction,
     skyvern_element: SkyvernElement,
 ) -> List[ActionResult]:
+    try:
+        current_text = await skyvern_element.get_attr("selected")
+        if current_text == action.option.label or current_text == action.option.value:
+            return [ActionSuccess()]
+    except Exception:
+        LOG.info("failed to confirm if the select option has been done, force to take the action again.")
+
     action_result: List[ActionResult] = []
     is_success = False
-    locator = skyvern_element.locator
+    locator = skyvern_element.get_locator()
```

Comment:
whoa seems like a big bug

Reply:
? What do you mean? I don't follow

Reply:
Sorry for the confusion! Did the old.locator implementation work? If not that's a huge bug! Nice find.

Reply:
Oh both of them could work. I just refactor it into the same format.